openpype/hosts/max/plugins/create/create_pointcache.py

```diff
@@ -9,3 +10,20 @@ class CreatePointCache(plugin.MaxCreator):
     label = "Point Cache"
     family = "pointcache"
     icon = "gear"
+
+    def create(self, subset_name, instance_data, pre_create_data):
+        instance_data["custom_attrs"] = pre_create_data.get(
+            "custom_attrs")
+
+        super(CreatePointCache, self).create(
+            subset_name,
+            instance_data,
+            pre_create_data)
+
+    def get_pre_create_attr_defs(self):
+        attrs = super().get_pre_create_attr_defs()
+        return attrs + [
+            BoolDef("custom_attrs",
+                    label="Custom Attributes",
+                    default=False),
+        ]
```

Comment:
I wonder if it'd make more sense to expose this as an Attribute Definition on the Extractors instead? Just so it's much closer to what it's related to. Thoughts?

e.g.

```python
class ExtractAlembic(pyblish.api.InstancePlugin,
                     publish.OpenPypePyblishPluginMixin):

    def process(self, instance):
        attr_values = self.get_attr_values_from_data(instance.data)
        custom_attrs = attr_values.get("custom_attrs", False)
        # do something with the value

    def get_attribute_defs(self):
        return [
            BoolDef("custom_attrs",
                    label="Custom Attributes",
                    default=False),
        ]
```